src/loki_alert_rules/error_level_logs_found.rule

```diff
@@ -0,0 +1,7 @@
+alert: ErrorLevelLogsFound
+expr: rate({%%juju_topology%%} |= "level=error"[1m]) > 0
```

Comment:
Wondering whether any error should be enough to trigger an alert or maybe if we should have a percentage-based check. So something like "if 5% of logs are errors, create an alert". What do you think?

Reply:
Following the same approach we did for Prometheus, having it as a percentage to leave a margin for error seems more reasonable